diracx-cli/tests/legacy/cs_sync/integration_test.cfg

```diff
@@ -9,21 +9,18 @@ DIRAC
   }
   Setups
   {
-    dirac-JenkinsSetup
-    {
-      Accounting = Production
-      Configuration = Production
-      DataManagement = Production
-      Framework = Production
-      Monitoring = Production
-      RequestManagement = Production
-      ResourceStatus = Production
-      StorageManagement = Production
-      Production = Production
-      Transformation = Production
-      WorkloadManagement = Production
-      Tornado = Production
-    }
+    Accounting = Production
+    Configuration = Production
+    DataManagement = Production
+    Framework = Production
```

Comment:
But why is this still needed ? There is no setup anymore, so we don't need this `<System> = Production` anymore, do we ?

Reply:
Ah, good point, this can be removed indeed.